compiler/src/dotty/tools/dotc/transform/Erasure.scala

```diff
@@ -642,7 +647,22 @@ object Erasure extends TypeTestsCasts{
                   }
 
                   val bridgeImplementations = minimalSet.map {
-                    sym => makeBridgeDef(member, sym)(ctx)
+                    sym =>
+                      if (member.symbol.is(Flags.Deferred)) {
+                        if (!sym.is(Flags.Deferred)) {
+                          // sometimes we need to forward an abstract method in a subclass to a non-abstract method in parent.
+                          // see https://github.com/lampepfl/dotty/issues/2072 for illustrations
+
+                          val d = member.symbol.copySymDenotation(initFlags = member.symbol.flags.&~(Flags.Deferred))
+                          d.installAfter(ctx.erasurePhase.asInstanceOf[Erasure])
+                          // Dark magic. Don't look here. 
```

Comment:
> Also do you need both installAfter and transformAfter ?

Yes. The comment here would entirely repeat all the history of transformAfter. I don't think it's worth simplifying it here. To understand why it's needed here one need to read the entire description of 979ee0ff41c2a957b22e886ea43ad4f070a9777d which introduces transformAfter